### ISSUE TITLE
About the configs of `MAML` and `MAML++`

### ISSUE THREAD

**Issue by WannaSeaU** [AntreasAntoniou/HowToTrainYourMAMLPytorch#29]
I noticed that `MAML` and `MAML++` configs for mini-imagenet differ in the term of `"total_epochs_before_pause"` (100 for the former and 101 for the latter), I'm wondering what this config means and why they are different. The `"total_epochs_before_pause"` configs of omniglot seem to be identical.

Also, I also noticed the config `"first_order_to_second_order_epoch"` is set to be -1, seems to be a conflict to the `Derivative-Order Annealing` strategy.

Thanks in advance!

**Comment by WannaSeaU** [comment]
Also, I noticed that MAML++ configs for mini-ImageNet don't use `Cosine Annealing of Meta-Optimizer Learning Rate` strategy since the min-learning rate is set the same as the initial learning rate.

**Comment by AntreasAntoniou** [comment]
"total_epochs_before_pause" is a hyperparameter that controls how often
python will exit. I used it mainly because I sometimes had to break my
experiments in multiple chunks due to compute limitation on clusters.

The first_order_to_second_order_epoch is overall useful at speeding things
up and is used at various experiments. However, in some other cases you
might converge faster without it. Hence why I left it like that in some
cases.

**Comment by AntreasAntoniou** [comment]
That's because in certain experiments you can converge faster without the
cosine annealer. I provided the exact configs to reproduce the results I
have in the paper.

**Comment by WannaSeaU** [comment]
Thanks! I've read all the codes, and it benefits me a lot! I can reproduce the results in the paper with the configs you provided. Awesome work!
